### ISSUE TITLE
Option to only attempt API connection manually

### ISSUE THREAD

**Issue by JTJutajoh** [isekaidev/stable.art#20]
It would be nice to have the option to have the plugin only attempt to connect to the webui API when I manually tell it to. Such as a "Connect" button under the Endpoint text box. 

I use Photoshop frequently for things other than just stable diffusion, but if I open Photoshop while the webui is not running, I get a disruptive error telling me that it failed to connect. I would prefer if it would not even try connecting unless I tell it to. 
Perhaps it could have a checkbox for automatically connecting so that users who only use Photoshop for this plugin can continue using it like before.

**Comment by DarkAlchy** [comment]
Yes, I booted mine up to do some things without SD being loaded and that popup hit me.  Annoying, so maybe a checkbox in settings to have it or turn it off forever would be nice.

**Comment by isekaidev** [comment]
Changed the logic of connecting - if your endpoint was inserted before then the plugin will only try to connect after clicking the 'Generate' button. If you do not have an endpoint, then it will try to connect after inserting it. The same is for models, so #19 is fixed too.
